test/development/acceptance-app/ReactRefreshLogBox.test.ts

```diff
@@ -415,7 +571,7 @@ describe.each(['default', 'turbo'])('ReactRefreshLogBox app %s', () => {
               ) as any
           ).href
       )
-    ).toMatchSnapshot()
+    ).toMatchInlineSnapshot(`"https://nextjs.org/"`)
```

Comment:
Ideally, we'd automatically include link testing in the snapshot. There are multiple ways to go about it and since we haven't broken that behavior, I'm not too worried we need to test it just yet.